spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/RetryableLoadBalancerExchangeFilterFunction.java

```diff
@@ -66,20 +66,23 @@
 	private static final Log LOG = LogFactory.getLog(RetryableLoadBalancerExchangeFilterFunction.class);
 
 	private static final List<Class<? extends Throwable>> exceptions = Arrays.asList(IOException.class,
-			TimeoutException.class,
-			org.springframework.cloud.client.loadbalancer.reactive.RetryableStatusCodeException.class);
+			TimeoutException.class, RetryableStatusCodeException.class);
 
 	private final LoadBalancerRetryPolicy retryPolicy;
 
 	private final LoadBalancerProperties properties;
 
 	private final ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory;
 
+	private final List<LoadBalancerClientRequestTransformer> transformers;
+
 	public RetryableLoadBalancerExchangeFilterFunction(LoadBalancerRetryPolicy retryPolicy,
-			ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory, LoadBalancerProperties properties) {
+			ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory, LoadBalancerProperties properties,
+			List<LoadBalancerClientRequestTransformer> transformers) {
```

Comment:
We cannot modify constructors after the major GA release. We can only add an additional constructor.

Reply:
Thanks, let's also deprecate the old one.

Reply:
Deprecated in b23f05fe5afc40a7dfa31fdccb39ffc819872079